src/Gui/ViewProviderLink.cpp

```diff
@@ -1883,7 +1883,7 @@ void ViewProviderLink::updateDataPrivate(App::LinkBaseExtension *ext, const App:
             if(canScale(v))
                 pcTransform->scaleFactor.setValue(v.x,v.y,v.z);
             SbMatrix matrix = convert(ext->getTransform(false));
-            linkView->renderDoubleSide(matrix.det3() < 0);
+            linkView->renderDoubleSide(matrix.det3() < 1e-7);
```

Comment:
Why this "magic" number and not
`Precision::Confusion()`
or a multiple of Precision::Confusion()

If not Precision::Confusion(), there should be a comment about the number

Reply:
Because it is the Gui core module, which has no dependency of OCC. @wwmayer Shall we add something like App::Confusion() constant?

Reply:
> Shall we add something like App::Confusion() constant?

I once had a discussion with openBrain IIRC and we came to the conclusion to copy OCC's Precision class to Base and he wanted to make a PR. Obviously he forgot about it.

Reply:
@0penBrain , can you please make the mentioned PR?

For now, meaning FC 0.20.1 I propose to merge the PR as it is so that I can backport the fix. The future PR won't be backportable.
OK?

Reply:
> @0penBrain , can you please make the mentioned PR?

Probably not in the coming weeks. Time for vacations. :)

Reply:
Found the thread where it was discussed: https://forum.freecadweb.org/viewtopic.php?f=10&t=65959
But openBrain didn't explicitly make an offer to provide a PR. Anyway here it is: bd5ef26536

Reply:
Many thanks!

So I will merge this PR, make the backport and @realthunder will make a PR only for master using the new Precision class.